spec/models/service_template_ansible_playbook_spec.rb

```diff
@@ -225,19 +225,18 @@
       expect(service_template.resource_actions.first.dialog.id).to eq new_dialog_record.id
     end
 
-    it 'uses the existing dialog if :service_dialog_id is passed in' do
+    it 'uses the existing dialog if :dialog_id is passed in' do
```

Comment:
@bzwei - It would also be good to cover the case where `:new_dialog_name` & `:dialog_id` were passed in.